app/views/layouts/application.html.erb

```diff
@@ -91,11 +91,11 @@
      background: #BABABA;
     }
 
-  </style>  
-  
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-slider/10.2.0/bootstrap-slider.js"></script>
+  </style>
+
+  <script defer src="https://cdnjs.cloudflare.com/ajax/libs/bootstrap-slider/10.2.0/bootstrap-slider.min.js"></script>
```

Comment:
Hmm, i'm just trying to think of where this is used so that it's clear it'll still work! 

See how you can use the "blame" view to find out: https://github.com/publiclab/plots2/blame/master/app/views/layouts/application.html.erb#L95

Given it links here: https://github.com/publiclab/plots2/pull/4633

Could you test that function to be sure, and just note that in this PR? That'll give us extra confidence! But this does look good to go!